External/Plugins/HaXeContext/Context.cs

```diff
@@ -577,6 +580,21 @@ public override void RemoveClassCompilerCache()
         }
         #endregion
 
+        #region SDK
+        private InstalledSDK GetCurrentSDK()
+        {
+            return hxsettings.InstalledSDKs.FirstOrDefault(sdk => sdk.Path == currentSDK);
```

Comment:
And I thought you were some C# god! ;)
Maybe I'll give another look at Resharper but I really didn't like it last time I tried.